rmgpy/rmg/model.py

```diff
@@ -173,6 +173,7 @@ class CoreEdgeReactionModel:
     `network_count`            A counter for the number of pressure-dependent networks created
     `index_species_dict`       A dictionary with a unique index pointing to the species objects
     `solvent_name`             String describing solvent name for liquid reactions. Empty for non-liquid estimation
+    `liquid_volumetric_mass_transfer_coefficient_power_law` Power law for estimating kLA
```

Comment:
Do we really need this on both RMG and CoreEdgeReactionModel?

Reply:
Yeah, we actually need that to be on both RMG and CoreEdgeReactionModel, analogous to how we deal with `solvent` and `kinetics_estimator`. The attribute on RMG is used during the function `read_input_file` to record the input specifications, and it passes the content to the attribute on CoreEdgeReactionModel in the function `load_input`. The attribute on CoreEdgeReactionModel is the one actually being used to generate data and attach to species.

Reply:
In the new push I remove this from CoreEdgeReactionModel